Client/Components/Custom.razor.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System.Net.Http.Json;
+using System.Reflection.Metadata.Ecma335;
+using System.Runtime.InteropServices;
+using API;
+using Client.API;
+using Microsoft.AspNetCore.Components;
+
+namespace Client.Components
+{
+    public partial class Custom
+    {
+        [Inject]
+        private HttpClient httpClient { get; set; }
+
+        [Inject]
+        private IApiHandler apiHandler { get; set; }
+
+        private string selectedHome = "";
+
+        private string selectedOther = "";
+
+        private Country _country { get; set; }
```

Comment:
Is the get and set needed for these or could they just be private instance variables?